app-frontend/src/app/pages/projects/edit/edit.html

```diff
@@ -10,15 +10,6 @@
       <i class="icon-project"></i>
       <span>Scenes</span>
     </a>
-    <a ui-sref="projects.edit.colormode"
```

Comment:
do we actually not use color modes anymore? I feel like we should allow color mapping still, since that serves a pretty clear, functional use case which isn't a bear to support (it's also not a mess like advanced color correction)

Reply:
No one uses it: https://azavea.slack.com/archives/D1EH808UU/p1570629213000200

And it's pretty easy to bring back if we decide that it's important for something.

Reply:
Woops that's a link to a private message. Anyway, it's used only in demo projects it seems